package.json

```diff
@@ -20,13 +27,22 @@
   "bugs": {
     "url": "https://github.com/surikaterna/kuery/issues"
   },
+  "prettier": "prettier-config-surikaterna",
   "devDependencies": {
-    "istanbul": "^0.3.2",
-    "jshint": "^2.5.10",
-    "mocha": "^2.0.1",
-    "should": "^4.1.0",
-    "eslint": "^2.5.3",
-    "eslint-config-surikat": "^1.1.0"
+    "@swc/cli": "^0.1.57",
+    "@swc/core": "^1.2.220",
+    "@swc/jest": "^0.2.22",
+    "@types/jest": "^28.1.6",
+    "@types/lodash": "^4.14.182",
```

Comment:
This version differs from the installed `lodash` version. Try to update the `lodash version and verify that it still works. If it doesn't work, decrease this version to be `@types/lodash@4.7`.